js/settings.js

```diff
@@ -2,22 +2,25 @@
 
 var fireballSize = 22;
 
-var getFireballSpeed = function (isMovingLeft) {
-  return isMovingLeft ? 2 : 5;
+var getFireballSpeed = (isMovingLeft) => {
+  const LEFT_SPEED = 2;
+  const RIGHT_SPEED = 5;
+  return isMovingLeft ? LEFT_SPEED : RIGHT_SPEED;
 };
 
 var wizardSpeed = 3;
 
 var wizardWidth = 70;
 
-var getWizardHeight = function () {
-  return 1.337 * wizardWidth;
+var getWizardHeight = () => {
+  const COEFFICIENT = 1.337;
+  return COEFFICIENT * wizardWidth;
 };
 
-var getWizardX = function (gameFieldWidth) {
+var getWizardX = (gameFieldWidth) => {
   return (gameFieldWidth - wizardWidth) / 2;
 };
```

Comment:
```suggestion
var getWizardX = (gameFieldWidth) => {
  return (gameFieldWidth - wizardWidth) / 2;
};
```
```suggestion
var getWizardX = (gameFieldWidth) => (gameFieldWidth - wizardWidth) / 2;
```

можно писать в сокращенном варианте